order.go

```diff
@@ -0,0 +1,108 @@
+package apiutil
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+var (
+	// OrderASC defines the ascending order keyword
+	OrderASC = "asc"
+	// OrderDESC defines the descending order keyword
+	OrderDESC = "desc"
+)
+
+// OrderBy represents ordering information
+type OrderBy struct {
+	OrderItems []Order
+}
```

Comment:
love the `OrderBy` name. I think this type should be defined on the slice of items:

```go
// OrderBy represents ordering information
type OrderBy []Order
```